### PR TITLE
Adding v0 rendering functionality 

### PR DESCRIPTION
**Pull request by ArashAhmadian**
Adding rendering funtionality implemented in `pygame` . This is something to start us off and get us going -- can add more features / fancy visuals after discussing options.

### CODE REVIEW

**Comment by Thanoshan** [comment]
What is .anim_consts? I'm getting an import error on that when trying to run this

**Comment by ArashAhmadian** [comment]
@Thanoshan oops forgot to push it

File: src/nepiada/env/nepiada.py [hunk 1]
```diff
@@ -91,8 +91,7 @@ def render(self):
             return
         elif self.render_mode == "human":
             # Temporary to print grid for debug purposes until we have a better way to render.
```
Comment (Thanoshan) [inline src/nepiada/env/nepiada.py:93]: nit: Remove this comment

File: src/nepiada/utils/grid.py [hunk 2]
```diff
@@ -58,11 +73,4 @@ def move_drone(self, agent, action):
...
-        return 0
-
-    ## For debug
-    def render_grid(self):
-        print("---------------")
-        for row in self.state:
-            print(row)
-        print("---------------")
\ No newline at end of file
+        return 0
```
Comment (Thanoshan) [inline src/nepiada/utils/grid.py:76]: nit: Can delete the comment on lines 69-70

**Review by Thanoshan — APPROVED** [review]
LGTM. A couple nit comments from me, but otherwise good to go.

**Comment by pandyah5** [comment]
If this PR has been reviewed can you please fix the conflicts and merge it? Thanks

**Pull request merged**